api/controllers/items.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../models');
+const { Item } = db;
+
+//    GET    /items
+//    POST   /items
+//    GET    /items/:id
+//    PUT    /items/:id
+//    DELETE /items/:id
+
+router.get('/', (req, res) => {
+  Item.findAll({}).then((items) => res.json(items));
+});
+
+router.post('/', (req, res) => {
+  let { content } = req.body;
+
+  Item.create({ content })
```

Comment:
I don't if you can directly do this. Try to insert a new row into the database and see if the database has that row.

Reply:
Good catch; fixed it.